API.php

```diff
@@ -1461,7 +1461,10 @@ public function importContainerVersion($exportedContainerVersion, $idSite, $idCo
             throw new Exception(Piwik::translate('TagManager_ErrorContainerVersionDoesNotExist'));
         }
 
-        $exportedContainerVersion = Common::unsanitizeInputValue($exportedContainerVersion);
+        if (!$isDraftRestoreCall) {
+            $draft = $this->exportContainerVersion($idSite, $idContainer);
+            $exportedContainerVersion = Common::unsanitizeInputValue($exportedContainerVersion);
```

Comment:
Why is this line necessary when it wasn't before? Is this fixing an additional pre-existing issue?